client/data/onboarding/resolvers.ts

```diff
@@ -0,0 +1,32 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { apiFetch, dispatch } from '@wordpress/data-controls';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { NAMESPACE } from '../constants';
+import { updateBusinessTypes } from './actions';
+import { Country } from './types';
+
+/**
+ * Retrieve business types from the site's REST API.
+ */
+export function* getBusinessTypes(): unknown {
```

Comment:
Add return type here.

Reply:
I think we should go with `unknown` for now. We will need types definitions from `@wordpress/data-controls` to properly type resolvers generators.